src/pages/LandmarkDetailScreen.tsx

```diff
@@ -111,7 +111,7 @@ export function LandmarkDetailScreen({ match }: UserDetailPageProps) {
             <IonCheckbox
               value={landmarkDetails.visited}
               checked={landmarkDetails.visited}
-              onIonChange={(e) => handleCheckBoxChange(e.target.checked)}
+              onIonChange={(e) => handleCheckBoxChange(e.target.checked, landmarkDetails?.inTrip)}
```

Comment:
I don't think the ? is required here. Other than that it looks good.

Reply:
Right! Removed it